pennylane/transforms/core/transform_program.py

```diff
@@ -109,3 +109,55 @@ def is_empty(self):
             bool: Boolean, True if empty, False otherwise.
         """
         return len(self) == 0
+
+    def is_informative(self):
+        """Check if the transform program is informative or not.
+
+        Returns:
+            bool: Boolean, True if empty, False otherwise.
+        """
+        return self.get_last().is_informative
+
+    def __call__(self, tapes):
+        processing_fns_list = []
+        classical_cotransforms_list = []
+
+        for transform_container in self:
+            num_tapes = len(tapes)
+            transform, args, kwargs, cotransform, _ = transform_container
+
+            execution_tapes = []
+            fns = []
+
+            for tape in tapes:
+                new_tapes, fn = transform(tape, *args, **kwargs)
+                execution_tapes.extend(new_tapes)
+                fns.append(fn)
+
+            new_num_tapes = len(new_tapes)
+
+            # Merge the processing function into in a single one
+            def processing_fn(
+                res,
+                num_tapes=num_tapes,
+                new_num_tapes=new_num_tapes,
+                p_fns=tuple(fns),
+            ):  # pylint: disable=cell-var-from-loop
+                final_results = [
+                    p_fns[idx](res[idx * new_num_tapes : (idx + 1) * new_num_tapes])
+                    for idx in range(num_tapes)
+                ]
+                return final_results
+
+            processing_fns_list.append(processing_fn)
+
+            # Merge the cotransform functions into in a single one
+            if transform_container.classical_cotransform is None:
+                classical_cotransforms_list.append(None)
+            else:
+                # TODO: coverage when add gradient transform
+                classical_cotransforms_list.append(cotransform)  # pragma: no cover
```

Comment:
```suggestion
            classical_cotransforms_list.append(cotransform)
```

Could we amend what a transform container returns to get rid of the logical branch here? 

What will the `cotransform` variable be if when `transform_container.classical_cotransform is None`?